test/test_testing.py

```diff
@@ -2782,6 +2782,77 @@ def test_add_layer(self):
     summary: Serv
 ''', plan.to_yaml())
 
+    def test_add_layer_merge(self):
+        client = self.get_testing_client()
+        plan = client.get_plan()
+        self.assertIsInstance(plan, pebble.Plan)
+        self.assertEqual('{}\n', plan.to_yaml())
+        client.add_layer('foo', pebble.Layer('''\
```

Comment:
textwrap.dedent() to be able to keep the indentation level of the multiline string

Reply:
done